src/_nebari/stages/infrastructure/__init__.py

```diff
@@ -315,7 +315,7 @@ class GCPNodeGroup(schema.Base):
 
 
 DEFAULT_GCP_NODE_GROUPS = {
-    "general": GCPNodeGroup(instance="e2-highmem-4", min_nodes=1, max_nodes=1),
+    "general": GCPNodeGroup(instance="e2-highmem-4", min_nodes=1, max_nodes=5),
```

Comment:
Can you update Azure max nodes to 5 to be consistent with the other cloud providers?

Reply:
Done.